akka-http-core/src/main/scala/akka/http/impl/util/LogByteStringTools.scala

```diff
@@ -23,7 +23,7 @@ private[akka] object LogByteStringTools {
   def logByteString(name: String, maxBytes: Int = MaxBytesPrinted): Flow[ByteString, ByteString, NotUsed] =
     Flow[ByteString].log(name, printByteString(_, maxBytes))
 
-  def logTLSBidi(name: String, maxBytes: Int = MaxBytesPrinted): scaladsl.BidiFlow[SslTlsOutbound, SslTlsOutbound, SslTlsInbound, SslTlsInbound, NotUsed] =
+  def logTLSBidi(name: String, maxBytes: Int = MaxBytesPrinted): BidiFlow[SslTlsOutbound, SslTlsOutbound, SslTlsInbound, SslTlsInbound, NotUsed] =
```

Comment:
Usually don't remove`scaladsl.` as you go please, clears up for readability esp. if in `impl`